.cursor/rules/sentry.mdc

```diff
@@ -0,0 +1,133 @@
+---
+description: Use this rule for integrating and configuring Sentry error tracking in Next.js projects. It helps set up Sentry correctly, capture server/client errors, and follow best practices for monitoring and debugging in production.
+globs: 
+alwaysApply: false
+---
+These examples should be used as guidance when configuring Sentry functionality within a project.
+
+# Exception Catching
+
+- Use `Sentry.captureException(error)` to capture an exception and log the error in Sentry.
+- Use this in try catch blocks or areas where exceptions are expected
```

Comment:
```suggestion
- Use this in try catch blocks or areas where exceptions are expected and not rethrown
```

Otherwise you'll get double-reporting as errors will be handled at the top level as well (or at least should be setup like that)